experimental/segmenter/src/lstm_bies.rs

```diff
@@ -168,3 +169,94 @@ impl Lstm {
         bies
     }
 }
+
+#[cfg(test)]
+mod tests {
```

Comment:
question: I assume these tests within this `mod` are adapted from `lstm_test.rs` without any change to the logic. If not, please let me know.

Reply:
`lstm_structs` and `'lstm_bies' are private module now. So I move `lstm_test.rs` to to `lstm_bies.rs`.